lib/internal/bootstrap/realm.js

```diff
@@ -146,16 +137,7 @@ const experimentalModuleList = new SafeSet();
 
   process.binding = function binding(module) {
     module = String(module);
-    // Deprecated specific process.binding() modules, but not all, allow
-    // selective fallback to internalBinding for the deprecated ones.
     if (processBindingAllowList.has(module)) {
-      if (runtimeDeprecatedList.has(module)) {
-        runtimeDeprecatedList.delete(module);
-        process.emitWarning(
-          `Access to process.binding('${module}') is deprecated.`,
-          'DeprecationWarning',
-          'DEP0111');
-      }
```

Comment:
Doesn't this do the opposite of runtime deprecating?

Reply:
This was emitting a warning whenever a runtime deprecated module is called with process.binding. Example, process.binding('async_wrap'). Now, all the modules are deprecated, so I removed them in favor of:

```js
process.binding = deprecate(process.binding,
                              'process.binding() is deprecated. ' +
    'Please use public APIs instead.', 'DEP0111');
```

Otherwise, users were going to see two warning deprecations for the same operation.